sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 4.2.0-beta.2 (Unreleased)
 
+- Removed `dist-browser` from the output folders. To bundle the Azure SDK libraries, please read our bundling guide: [link](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md).
```

Comment:
```suggestion
- [Breaking] Removed `dist-browser` from the published package. To bundle the Azure SDK libraries, please read our bundling guide: [link](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md).
```

Reply:
@jeremymeng @bterlson

Reply:
I've added this feedback. I won't resolve this though, since you're bringing this to the attention of others! I want them to see this before this gets merged.